packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -122,7 +122,7 @@ const MobileRowRenderer = ({ row, is_footer, columns_map, server_time, onClickCa
     );
 };
 
-const OpenPositionsTable = ({
+export const OpenPositionsTable = ({
```

Comment:
Why are you exporting this component? it is not being used outside of this file!

Reply:
This export is required for tests